src/lib/broadcast.jl

```diff
@@ -164,31 +164,31 @@ end
 # Avoid hitting special cases for `Adjoint` etc.
 _broadcast(f::F, x...) where F = materialize(broadcasted(f, x...))
 
-_get(x::Tuple, i) = x[i]
-_get(::Nothing, i) = nothing
 collapse_nothings(xs::AbstractArray{Nothing}) = nothing
 collapse_nothings(xs) = xs
 
-@adjoint function broadcasted(::AbstractArrayStyle, f, args...)
+@adjoint function broadcasted(::AbstractArrayStyle, f::F, args...) where {F}
```

Comment:
Not sure I meant to commit this! Was trying to see if any trick might solve the `identity∘tanh` case.